libs/designer/src/lib/core/templates/utils/helper.ts

```diff
@@ -100,3 +101,9 @@ export const getConnectorResources = (intl: IntlShape) => {
     }),
   };
 };
+
+export const useExistingWorkflowNames = () => {
+  return useQuery(['getExistingWorkflowNames'], async () => {
+    return await TemplateService()?.getExistingWorkflowNames();
+  });
+};
```

Comment:
This should be moved to queries folder to a template.ts file, like how we have it structured for designer. Not blocking this pr